### DISCUSSION TITLE
Contact Organisation owner

### DISCUSSION THREAD

**Discussion by xpessoles** [community/community#112269] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, 
I am the owner of an organisation, but I have lost the id /password to manage it. 
How can I find the address ?
Is there a contact page to contact the manager ? 
Best regards, 
X

**Comment by jsoref** [comment]
Organizations can have 1 or more owners, anyone who is an owner can log in and visit https://github.com/organizations/ORGANIZATION_NAME/settings/profile to edit it.

For anything else, you'd need to talk to [support](https://support.github.com)
